<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QWidget" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1071</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="no_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="no">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Student no :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string> INFORMATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;kl&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 8pt &quot;MS Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Selected Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Course Details</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>931</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string>Student Information </string>
     </property>
     <widget class="QLabel" name="label_bolum">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_no">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>165</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>120</width>
        <height>86</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NAME :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Department :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ad">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="no2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>120</width>
        <height>86</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>student No :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CGPA :</string>
      </property>
     </widget>
     <widget class="QLabel" name="cgpa">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>50</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>122</width>
        <height>88</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>0</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>841</width>
       <height>561</height>
      </rect>
     </property>
     <property name="title">
      <string>Courses</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>801</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>COURSE ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SECTION ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SEMESTER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>YEAR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GRADE</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>1011</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>COURSE ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SECTION ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SEMESTER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>YEAR</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BUILDING</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ROOM NO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time Slot Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CREDIT</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="scourse">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses :</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
